### PR TITLE
Layers side panel

### PR DESCRIPTION
**Pull request by TAnas0**
## What is the Purpose?
Side panel for the Tables/Data view

## What was the approach?
Created a Drawer for the side panel of Data to list layers

## Are there any concerns to addressed further before or after merging this PR?
This is still Work In Progress

## Mentions?
@ninetteadhikari @sannleen @michaelbukachi 

## Issue(s) affected?
#156 #118

### CODE REVIEW

**Comment by andrewtpham** [comment]
@TAnas0 the current draft looks great, well done! 💪 

A few points to keep in mind is that we will want to make the list of layers scrollable in case a table has more layers than can be listed on the drawer. Also, you may want to see if you can get the `+ Add new layer` button to stick to the bottom of the drawer when you minimize the screen.

**Comment by andrewtpham** [comment]
@TAnas0 it looks great! 🙌 I push a few more styling changes to the switch and more icon. Could you help me resolve the conflict ☝️ 

Also, this is to note that when the page weight is below 700 px, the layer list starts to show under the `+ Add New Layer` button. If this is not an easy fix, we can always address later :) 
image

File: src/views/tables/index.vue [hunk 1]
```diff
@@ -1,6 +1,46 @@
...
+        >
+          <div>{{ layer.title }}</div>
+          <div style="display: flex">
+            <h-switch width="35" />
+            <h-dropdown>
+              <div slot="title">
+                <i class="el-icon-more el-icon--right" />
+              </div>
+              <template slot="items">
+                <el-dropdown-item>Edit layer</el-dropdown-item>
```
Comment (ninetteadhikari) [inline src/views/tables/index.vue:24]: don't we need this 'Edit layer' to be in the translation file? same for 'Delete layer' and 'Add new layer'... I might be wrong 🙃

Comment (TAnas0) [reply src/views/tables/index.vue:24]: Yes, I didn't account for i18n

**Pull request merged**